src/main/resources/com/google/api/codegen/ruby/main.snip

```diff
@@ -228,8 +222,10 @@
         scopes: ALL_SCOPES,
         client_config: {},
         timeout: DEFAULT_TIMEOUT,
-        app_name: "gax",
-        app_version: Google::Gax::VERSION
+        app_name: nil,
+        app_version: nil,
+        lib_name: "gapic",
+        lib_version: "0.1.0"
```

Comment:
Update: After looking at the spec again, I think I am wrong about the first bullet (the Python implementation is specifically called out as expected to be slightly different from all the others).